src/graphics/engine/engine.cpp

```diff
@@ -431,7 +431,7 @@ bool CEngine::ProcessEvent(const Event &event)
     {
         auto data = event.GetData<KeyEventData>();
 
-        if (data->key == KEY(F12))
+        if (data->key == KEY(F11))
```

Comment:
I would make this handle both cases, so that people who are used to pressing F12 can keep doing it

Reply:
You mean using #ifdef? I'm not sure if keys should change depending on the build, it would seem confusing to most people in my opinion.

Reply:
No, I mean just `data->key == KEY(F11) || data->key == KEY(F12)` 😛